### PR TITLE
feat: :heavy_plus_sign: install and test tailwind

### PR DESCRIPTION
**Pull request by MihlaliMgwebi**
I set up my directory structure, installed tailwind and tested my installation.

After request for changes or approval, I will build my first component.

### CODE REVIEW

File: github-by-mish/package.json [hunk 1]
```diff
@@ -16,7 +16,10 @@
     "@types/react": "^18.0.27",
     "@types/react-dom": "^18.0.10",
     "@vitejs/plugin-react": "^3.1.0",
+    "autoprefixer": "^10.4.13",
```
Comment (kphutiyagae) [inline github-by-mish/package.json:19]: Is autoprefixer necessary?

Comment (kphutiyagae) [reply github-by-mish/package.json:19]: and postcss?

**Review by kphutiyagae — CHANGES_REQUESTED** [review]

File: github-by-mish/package.json (hunk 1 above)
Comment (MihlaliMgwebi) [reply github-by-mish/package.json:19]: Not at all. I have removed them and pushed my changes. 

Please review again. Thank you for the comment

Comment (kphutiyagae) [reply github-by-mish/package.json]: Noted

**Review by kphutiyagae — APPROVED** [review]
Looks good, proceed to merge!

**Pull request merged**